daemon/prune.go

```diff
@@ -259,20 +256,20 @@ func (daemon *Daemon) clusterNetworksPrune(pruneFilters filters.Args) (*types.Ne
 
 // NetworksPrune removes unused networks
 func (daemon *Daemon) NetworksPrune(pruneFilters filters.Args) (*types.NetworksPruneReport, error) {
-	rep := &types.NetworksPruneReport{}
-	clusterRep, err := daemon.clusterNetworksPrune(pruneFilters)
-	if err != nil {
-		logrus.Warnf("could not remove cluster networks: %v", err)
-	} else {
-		rep.NetworksDeleted = append(rep.NetworksDeleted, clusterRep.NetworksDeleted...)
+	if _, err := getUntilFromPruneFilters(pruneFilters); err != nil {
```

Comment:
Hmf, missed your ping. Yes I was a bit in doubt here; currently, it's just the `until` value that's used, however, possibly more filters are added in future, in which case we most likely want to pass those as `pruneFilters`.

_Ideally_, I think "cluster" and "local" networks should be treated the same, instead of needing two separate ways to loop / delete them